tabpfn_client/service_wrapper.py

```diff
@@ -58,7 +60,14 @@ def set_token_by_login(self, email: str, password: str) -> tuple[bool, str]:
         self.set_token(access_token)
         return True, message
 
-    def try_reuse_existing_token(self) -> bool:
+    def get_user_email_verification_status(
+        self, email: str, access_token_required: bool = False
+    ) -> tuple[bool, str]:
+        return self.service_client.get_user_email_verification_status(
+            email, access_token_required
+        )
+
+    def try_reuse_existing_token(self) -> bool or (bool, str):
```

Comment:
you have to use the `|` to say or in type strings.
But this union return types (that is what they are called) should be used as little as possible, as it is hard to understand.

Reply:
Sure

Reply:
Done